packages/ia-components/sandbox/tiles/TileComponent.js

```diff
@@ -5,16 +5,27 @@ import IAReactComponent from '../IAReactComponent'; // Encapsulates differences
 //import PropTypes from 'prop-types'
 //TODO-IAUX need to standardise API as this uses the "ArchiveMemberSearch" class to provide necessary details for the Tile.
 //import ArchiveMemberSearch from "@internetarchive/dweb-archivecontroller/ArchiveMemberSearch";
-import Util from "../../Util"; //TODO-IAUX for mediatype_canonical and number_format which need porting to some kind of common libary between IAUX and dweb-archive
 import TileImage from "./TileImage";
 import ParentTileImg from "./ParentTileImg";
-//Both these fail - not compiling the JSX
-//import TileImage from "@internetarchive/ia-components/sandbox/tiles/TileImage";
-//import TileImage from "../../iaux/packages/ia-components/sandbox/tiles/TileImage";
+import AnchorDetails from "../AnchorDetails";
+
+
+function number_format(nStr) //this is just addCommas now
```

Comment:
love a good helper function. will keep note of this to extract in the future when we have a place for them to be used for all